base/workflow/R/create_execute_test_xml.R

```diff
@@ -22,8 +22,11 @@
 #'   (character) BETY database connection options. Default values for all of
 #'   these are pulled from `<pecan_path>/web/config.php`.
 #' @param db_bety_driver (character) BETY database connection driver (default = `"Postgres"`)
-#' @return
+#' @return A list with two entries:
+#'  * `sys`: Exit value returned by the workflow (0 for sucess).
+#'  * `outdir`: Path where the workflow results are saved
 #' @author Alexey Shiklomanov, Tony Gardella
+#' @importFrom dplyr %>% .data
```

Comment:
`.data` is originally from `rlang` and is re-exported by `dplyr`. Is it standard practice to import from packages that re-export functions? I can rationalize both approaches -- doing it this way reduces the length of our top-level dependencies list, and people are much more likely to know `dplyr` than `rlang`. On the other hand, importing the original function points you more directly to its documentation.

No need to change this here -- just curious about your thoughts on these strategies in the future.

Reply:
I wondered about this too -- I'm not sure what's standard, but if we can find out we should follow the standard. I'll ask Twitter and report back.

My reasoning right now is that (1) `dplyr` re-exports functions precisely to make them easy to use them inside dplyr as we are here; and that (2) if the dplyr-exported and rlang-exported versions of a function were to differ in any way, we can hope the version dplyr exports would be the version dplyr was most compatible with.

Reply:
I mean, that's exactly what happened to `devtools::install_deps` in the 1.0->2.0 interval, soooooo.